buildPackageInJenkins.sh

```diff
@@ -0,0 +1,33 @@
+# Remove any lingering containers from previously failed builds
+docker rm my-yarn --force || true
+docker rm my-composer --force || true
+
+# Run the Jasmine tests
+docker pull sugarcrmdev/school:yarn
+docker run -t -d --name my-yarn sugarcrmdev/school:yarn
+docker exec my-yarn bash
+docker exec my-yarn grunt test-js
+
+# Run the PHPUnit tests
+docker pull sugarcrmdev/school:composer
+docker run -t -d --name my-composer sugarcrmdev/school:composer
+docker exec my-composer bash
```

Comment:
(Similar to above) What is this line used for?

Reply:
I thought I needed these in order to be able to execute commands in the container.  I just did some testing, and I was wrong. Good catch!